core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -9,7 +9,7 @@
 
 <md-card ng-class="[isCollectionPreviewTile ? 'oppia-activity-summary-tile-mobile' : 'oppia-activity-summary-tile', {'small-width': !isWindowLarge}, {'oppia-activity-playlist-tile': isPlaylistTile()}]"
          style="position: relative; z-index: 5;" ng-if="!isRefresherExploration">
-  <a ng-href="<[getExplorationLink()]>" target="<[openInNewWindow ? '_blank' : '_self']>">
+  <a ng-href="<[getExplorationLink()]>" target="<[openInNewWindow ? '_blank' : '_self']>" class="protractor-mobile-test-exploration">
```

Comment:
1) Same as above, if it is visible on Desktop, should just be `protractor-test-exploration`

2) This <a> tags covers the entire md-card. Shouldn't it be more descriptive than just `exploration`? How about `exploration-summary-tile`?

3) Is there any other clickable protractor element on mobile? How are we clicking them on the desktop version?

Reply:
Done, thanks!